conf/ocs_version/ocs-4.6.yaml

```diff
@@ -1,6 +1,6 @@
 ---
 DEPLOYMENT:
-  default_ocs_registry_image: "quay.io/rhceph-dev/ocs-olm-operator:latest-4.6"
+  default_ocs_registry_image: "quay.io/rhceph-dev/ocs-bundle-operator:latest-4.6"
```

Comment:
The same here.

Also we still need to somehow dynamically update value from constants which I pointed out in different general comment in this PR.